File: 백준/Bronze/9085. 더하기/더하기.cc
#include <stdio.h>

int main() {
    int n, num, a, count=0;
    scanf("%d", &n);
    for(int i=0;i<n;i++){
            scanf("%d", &num);
            for(int y=0; y<num;y++){
                scanf("%d", &a);
                count += a;
            }
            printf("%d\n", count);
            count = 0;
        }
    
    return 0;
}

File: 백준/Bronze/9085. 더하기/더하기.java
import java.util.Scanner;

public class Main {
    public static void main(String args[]) {
        Scanner sc = new Scanner(System.in);
        int n, num, a, count = 0;
        n = sc.nextInt();
        for(int i = 0; i < n; i ++){
            num = sc.nextInt();
            for (int y = 0; y < num; y++){
                a = sc.nextInt();
                count += a;
            }
            System.out.printf("%d\n", count);
            count = 0;
        }
    }
}